test/alternator/test_streams.py

```diff
@@ -619,6 +637,8 @@ def fetch_more(dynamodbstreams, iterators, output):
 # Note that the order of events is only guaranteed (and therefore compared)
 # inside a single partition.
 def compare_events(expected_events, output, mode):
+    print('compare events output', output)
+    print('compare events expected', expected_events)
```

Comment:
Please consider if you really want to leave these printouts, or you just did it for your own debugging purposes. Next time you'll want to debug this code, you can add these printouts again, you probably don't need to leave it in.